connector_builder_mcp/_connector_builder.py

```diff
@@ -46,6 +46,12 @@ class StreamTestResult(BaseModel):
     message: str
     records_read: int = 0
     errors: list[str] = []
+    records: list[dict[str, Any]] | None = Field(
+        default=None, description="Actual record data when include_records=True"
+    )
+    slices: list[dict[str, Any]] | None = Field(
+        default=None, description="Raw slices data when include_raw_response_data=True"
```

Comment:
Use my description. You'll need to format though for line length. Pull before updating.

```suggestion
        default=None, description="By default, raw request and response data will be returned only upon error. Set to True or False, to always or never return the raw response data."
```